tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Reload/ProjectReloadInterceptorTests.cs

```diff
@@ -5,204 +5,203 @@
 using Microsoft.VisualStudio.ProjectSystem.Configuration;
 using StringReader = System.IO.StringReader;
 
-namespace Microsoft.VisualStudio.ProjectSystem
+namespace Microsoft.VisualStudio.ProjectSystem.VS;
+
+public class ProjectReloadInterceptorTests
 {
-    public class ProjectReloadInterceptorTests
-    {
-        [Theory]
-        [InlineData(
-            """
+    [Theory]
+    [InlineData(
+        """
             <Project>
               <PropertyGroup>
               </PropertyGroup>
             </Project>
             """,
-            """
+        """
             <Project>
               <PropertyGroup>
                 <TargetFrameworks>net45</TargetFrameworks>
               </PropertyGroup>
             </Project>
             """)]
-        [InlineData(
-            """
+    [InlineData(
+        """
             <Project>
               <PropertyGroup>
                 <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
                 <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
               </PropertyGroup>
             </Project>
             """,
-            """
+        """
             <Project>
               <PropertyGroup>
                 <TargetFrameworks>net45;netcoreapp2.2</TargetFrameworks>
               </PropertyGroup>
             </Project>
             """)]
-        [InlineData("""
+    [InlineData("""
             <Project>
               <PropertyGroup>
                 <TargetFrameworks>net45;netcoreapp2.2</TargetFrameworks>
               </PropertyGroup>
             </Project>
             """,
-            """
+        """
```

Comment:
Was this white space change intentional?

Reply:
Sort of. When I updated the namespace I switched to using file-scoped namespaces, and the IDE adjusted indentation. That seems suspicious, since it will affect the strings contents, though it doesn't matter in this particular case.